examples/ding-dong-bot.py

```diff
@@ -1,17 +1,35 @@
-#!/usr/bin/env python3.8
 """doc"""
 import asyncio
-
 from wechaty import Wechaty
+from wechaty.user import Message
+from wechaty_puppet import PuppetOptions
+from wechaty_puppet_hostie import HostiePuppet
+
+
+async def message(msg: Message):
+    """back on message"""
+    from_contact = msg.talker()
+    text = msg.text()
+    if from_contact is not None and text == 'ding':
+        await from_contact.say('dong')
+    print(msg)
+
+
+async def do_some_thing():
+    """do some thing"""
+    friends = await bot.Contact.find_all()
+    print(friends)
+    print('dong some thing')
+
+
+# puppet_options = PuppetOptions(token='your-token-here')
+hostie_puppet = HostiePuppet(PuppetOptions('your-token-here'), 'hostie-puppet')
+bot = Wechaty(hostie_puppet).on('message', message)
```

Comment:
Can we use the same argument as the TypeScript Wechaty?

```py
bot = Wechaty({ puppet: 'wechaty-puppet-hoswtie', puppetOptions: { token: 'the_token' } })
```

Reply:
Ok, I will refactor it later.